Forget password component

eng-somaia opened the pull request.

eng-somaia committed:
Feat: intial commit

eng-somaia committed:
Fix: initial commit

eng-somaia committed:
Feat: add forget password component

add forget PW in navbar temporarily. Need to be converted to a pop-up.

MuhannedNoman commented:
@eng-somaia I tried this and didn't receive a password ?

MuhannedNoman commented:
Nvm It worked it just took time.

MuhannedNoman reviewed (changes requested):
Remove the container since there is no need for that. 
Change the Component name to ResetPassword and make sure the folder name is following the same convention. 
Remove it form the Navbar. 
Display it in the Login for under the form.

eng-somaia committed:
Fix: delete & edit

delete ForgetPassword container, rename ResetPassword component and handle the corresponding changes, remove it from navbar and display it in the login container.

eng-somaia committed:
Fix: update test snapshots.

eng-somaia committed:
Fix: make reset pW as a modal.

eng-somaia committed:
Fix: edit & redo

resolve conflicts and remake reset PW model again (T_T) due to changes in Login file.

eng-somaia committed:
Fix: update test snapshots.

eng-somaia committed:
Fix: edit bug.

eng-somaia committed:
Fix: update test snapshots.
